Makefile

```diff
@@ -8,4 +8,6 @@ run:
 run-backend:
 	docker-compose up api worker
 upgrade:
-	git submodule update --remote
\ No newline at end of file
+	git submodule update --remote
+tunnel:
+	cd ./react-storefront/apps/saleor-app-checkout && npx saleor app tunnel 3001
```

Comment:
@witoszekdev this will fail if saleor-cli is not installed. I'm not sure if this is the best idea 🤔

Reply:
nope, it uses `npx` which installs saleor-cli - we use it in `react-storefront` docs

Reply:
We could also add `npm install -g` in `bootstrap` script, but to be honest it seems like even worse idea